### PR TITLE
Basket functionality

### PR DESCRIPTION
**Pull request by ahmedsatti101**

### CODE REVIEW

File: myapp/tests.py [hunk 1]
```diff
@@ -32,18 +32,6 @@ def test_product_type_choices(self):
...
-        expected_value = '+447123456789'
-        my_model = Customer(phone_number=expected_value)
-        actual_value = my_model.phone_number
-        self.assertEqual(expected_value, actual_value)
-        
-    def test_postcode_format(self):
-        expected_value = 'M20 2YH'
-        my_model = Customer(postcode=expected_value)
-        actual_value = my_model.postcode
-        self.assertEqual(expected_value, actual_value)
```
Comment (yasir-satti) [inline myapp/tests.py:46]: Why did you delete these tests ?

Comment (ahmedsatti101) [reply myapp/tests.py:46]: because i deleted the model for them so I dont need it

**Review by yasir-satti — APPROVED** [review]
